{
	"id": "oval:com.redhat.unaffected:def:20213514",
	"version": "635",
	"class": "vulnerability",
	"metadata": {
		"title": "Unaffected components for: CVE-2021-3514 389-ds-base: sync_repl NULL pointer dereference in sync_create_state_control() (moderate)",
		"reference": [
			{
				"ref_id": "CVE-2021-3514",
				"ref_url": "https://access.redhat.com/security/cve/CVE-2021-3514",
				"source": "CVE"
			}
		],
		"description": "Red Hat's versions of the associated software have been determined to NOT be affected by CVE-2021-3514.",
		"advisory": {
			"from": "secalert@redhat.com",
			"severity": "Moderate",
			"updated": {
				"date": "2022-04-17"
			},
			"cve": [
				{
					"text": "CVE-2021-3514",
					"cvss3": "6.5/CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
					"cwe": "CWE-476",
					"href": "https://access.redhat.com/security/cve/CVE-2021-3514",
					"impact": "moderate",
					"public": "20210401"
				}
			],
			"affected": {},
			"affected_cpe_list": {
				"cpe": [
					"cpe:/a:redhat:enterprise_linux:9",
					"cpe:/a:redhat:enterprise_linux:9::appstream",
					"cpe:/a:redhat:enterprise_linux:9::crb",
					"cpe:/a:redhat:enterprise_linux:9::highavailability",
					"cpe:/a:redhat:enterprise_linux:9::nfv",
					"cpe:/a:redhat:enterprise_linux:9::realtime",
					"cpe:/a:redhat:enterprise_linux:9::resilientstorage",
					"cpe:/a:redhat:enterprise_linux:9::sap",
					"cpe:/a:redhat:enterprise_linux:9::sap_hana",
					"cpe:/a:redhat:enterprise_linux:9::supplementary",
					"cpe:/o:redhat:enterprise_linux:9",
					"cpe:/o:redhat:enterprise_linux:9::baseos"
				]
			},
			"issued": {}
		},
		"affected": {}
	},
	"criteria": {
		"operator": "OR",
		"criterias": [
			{
				"operator": "AND",
				"criterias": [
					{
						"operator": "OR",
						"criterias": [
							{
								"operator": "AND",
								"criterions": [
									{
										"test_ref": "oval:com.redhat.cve:tst:20223517005",
										"comment": "python3-lib389 is installed"
									},
									{
										"test_ref": "oval:com.redhat.unaffected:tst:202035518002",
										"comment": "python3-lib389 is not installed"
									}
								]
							},
							{
								"operator": "AND",
								"criterions": [
									{
										"test_ref": "oval:com.redhat.cve:tst:20223517003",
										"comment": "389-ds-base-libs is installed"
									},
									{
										"test_ref": "oval:com.redhat.unaffected:tst:202035518004",
										"comment": "389-ds-base-libs is not installed"
									}
								]
							},
							{
								"operator": "AND",
								"criterions": [
									{
										"test_ref": "oval:com.redhat.cve:tst:20223517011",
										"comment": "389-ds-base is installed"
									},
									{
										"test_ref": "oval:com.redhat.unaffected:tst:202035518006",
										"comment": "389-ds-base is not installed"
									}
								]
							}
						]
					}
				],
				"criterions": [
					{
						"test_ref": "oval:com.redhat.cve:tst:20052541003",
						"comment": "Red Hat Enterprise Linux 9 is installed"
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:com.redhat.cve:tst:20052541004",
				"comment": "Red Hat Enterprise Linux must be installed"
			}
		]
	}
}
